examples/workflow/face_recognition_11/face_recognition_11/seed_test_suite.py

```diff
@@ -150,4 +150,10 @@ def main(args: Namespace) -> int:
         default=f"s3://{BUCKET}/{DATASET}/meta/metadata.csv",
         help="CSV file containing the metadata of each image. See default CSV for details.",
     )
+    ap.add_argument(
+        "--suite_name",
```

Comment:
Actually, want to flag that I think we've used `--test-suite-name` in the past. Probably worth it to be explicit that it's a test suite for kolena workflow.

Reply:
Quick grep indicates that other places use `--test-suite` or `--test_suites` as well -- not too consistent all over the place, but we should include `test` in the arg name.

Reply:
Sounds good! I'll loop back and raise a PR updating all them to be consistent